.nycrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Typing and reading JSON is great for computers, but less so for people. Can you use the YAML formatting for this?

Reply:
Yea I can easily do that.